runbot_travis2docker/models/runbot_repo.py

```diff
@@ -18,3 +19,27 @@ class RunbotRepo(models.Model):
         "If is empty won't push it. "
         "Don't Use this feature if you use just one runbot server.")
     travis2docker_test_disable = fields.Boolean('Test Disable?')
+    uses_weblate = fields.Boolean()
```

Comment:
We can remove this field because if we have a `weblate_token` so is a weblate_ok